rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -121,13 +118,11 @@ class Player : public rclcpp::Node
 
   /// \brief Advance player to the message with closest timestamp >= time_point.
   /// \details This is blocking call and it will wait until current message will be published
-  /// and message queue will be refilled. Could take substantial amount of time to find
-  /// required message or return player to the original stage.
+  /// and message queue will be refilled. If the given time stamp is after the end of the bag,
+  /// then the player will either end or loop consistent with player option settings.
```

Comment:
Can we clarify this language even more? I think it's 
* if time_point is before the beginning of the bag, then playback time will be set to the beginning of the bag
* if time_point is after the end of the bag, playback time will be set to the end of the bag, which will then end playback, or if loop is enabled then will start playing at the beginning of the next loop"

Reply:
ok